docs/contributing/coding_guide.rst

```diff
@@ -481,6 +481,118 @@ by an angular frequency to get a length scale:
 
 >>> d_i = (c/omega_pi).to(u.m, equivalencies=u.dimensionless_angles())    # doctest: +SKIP
 
+.. _aliases:
+
+Aliases
+=======
+
+An :term:`alias` is an abbreviated version of a commonly used function
+that is intended for interactive use. For example,
+`~plasmapy.formulary.speeds.va_` is an alias to
+`~plasmapy.formulary.speeds.Alfven_speed`.
```

Comment:
```suggestion
An :term:`alias` is an abbreviated version of a commonly used function.
For example, `~plasmapy.formulary.speeds.va_` is an alias to
`~plasmapy.formulary.speeds.Alfven_speed`.  It is intended to give
users the option for shortening their code while maintaining
readability and explicit meaning.  As such, :term:`aliases` are given
to functionality that already has a widely used symbol in the
community's literature.
```

[semi-optional]  The phrase "...intended for interactive use." threw me off because it does not seem to have a real meaning.  Trying to come up with something else, sent me down this rabbit hole.